### PR TITLE
Added notice about timezones in documentation on testing

### PR DESCRIPTION
**Pull request by carlbordum**
Hopefully the next dev won't waste time on this :)

### CODE REVIEW

File: doc/API-NOTES.rst [hunk 1]
```diff
@@ -115,6 +115,10 @@ Make sure the parameter ``moxrestfrontend.rest.host`` in
 ``agents/MoxRestFrontend/moxrestfrontend.conf`` is set to
 ``http://localhost:5000``.
 
+Make sure to set the local time to the same as Copenhagen when testing. This
```
Comment (danchr) [inline doc/API-NOTES.rst]: `Copenhagen` ➡️ `Europe/Copenhagen`.

File: doc/API-NOTES.rst [hunk 2]
```diff
@@ -115,6 +115,10 @@ Make sure the parameter ``moxrestfrontend.rest.host`` in
...
+can be done with the following command: ``cp
+/usr/share/zoneinfo/Europe/Copenhagen /etc/localtime``.
```
Comment (danchr) [inline doc/API-NOTES.rst]: Actually, `dpkg-reconfigure tzdata` is vaguely better IMO — but perhaps we should add a test to verify that this is indeed the case, or make the test check so in its `setUp` routine.

Comment (carlbordum) [reply doc/API-NOTES.rst]: Advantage by copying the file is that its easier to put in a shell script.

Do you want me to change it?

Comment (agger-magenta) [reply doc/API-NOTES.rst]: The correct/authorized way of doing this is:
```
sudo timedatectl set-timezone Europe/Copenhagen
```
That's better than copying files around yourself.

**Pull request merged**